pkg/dispers/metadata/metadata_test.go

```diff
@@ -0,0 +1,85 @@
+package dispers
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"testing"
+
+	"github.com/cozy/checkup"
+	"github.com/cozy/cozy-stack/pkg/config/config"
+	"github.com/cozy/cozy-stack/pkg/couchdb"
+	"github.com/cozy/cozy-stack/pkg/prefixer"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExecutionMetadata(t *testing.T) {
+
+	_, err := RetrieveExecutionMetadata("mean")
+	assert.Error(t, err)
+
+	// Let's Create A ExecutionMetadata
+	meta, err := NewExecutionMetadata("subscribe", "Cozy-DISPERS", url.URL{Scheme: "http"})
+	assert.NoError(t, err)
+
+	// Let's Create Task For The ExecutionMetadata
+	task, err := NewTaskMetadata()
+	assert.NoError(t, err)
+	err = meta.SetTask("Hope It Will Not Crash", task)
+	assert.NoError(t, err)
+
+	err = meta.EndExecution(errors.New("Oh nooooo!"))
+	assert.NoError(t, err)
+
+	// Let's Create A ExecutionMetadata
+	meta, err = NewExecutionMetadata("query", "mean", url.URL{Scheme: "http"})
+	assert.NoError(t, err)
+
+	meta2, err := RetrieveExecutionMetadata("mean")
+	assert.Equal(t, meta.ID(), meta2.ID())
+	assert.Equal(t, meta.Rev(), meta2.Rev())
+	assert.NoError(t, err)
+
+	// Let's Create Task For The ExecutionMetadata
+	task, err = NewTaskMetadata()
+	assert.NoError(t, err)
+	err = meta2.SetTask("Concept Indexor", task)
```

Comment:
It would be useful to have all the actors roles in a const (CI, TF, etc)

Reply:
Can I do this right now but you review it in another PR (enclaves-subscribe) ? I've got the perfect place to write consts for actors roles !

Reply:
Sure, no problem

Reply:
Just don't forget to update this test accordingly